### DISCUSSION TITLE
1.4.0rc4

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#18018] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.4.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.4.0rc4/

Git tag for the release:
https://github.com/apache/superset/tree/1.4.0rc4

The Change Log for the release:
https://github.com/apache/superset/blob/1.4.0rc4/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.4.0rc4/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.4.0rc4'>1.4.0rc4</a>.</em>

**Comment by eschutho** [comment]
The cherries added to this rc on top of rc3 were:
#17945 fix(dashboard): scope status of filter not update in dashboard metadata (@stephenLYZ)
#17349 fix(Dashboard): Check validity of control item (@geido)
#17842 fix(dashboard): update native filter info in metadata is not updated (@stephenLYZ)
#17964 chore: bump FAB to 3.4.3 (@dpgaspar)
#17894 chore: bump gunicorn to 20.1.0 (@mporracindie)
#17420 chore: Bump FAB to 3.4.0 (@kamalkeshavani-aiinside)

**Comment by monkey01127** [comment]
+1

**Comment by kamalkeshavani-aiinside** [comment]
+1
